.travis.yml

```diff
@@ -24,13 +24,15 @@ env:
   - REMOVE_XDEBUG="0"
   - REMOVE_XDEBUG="1"
 
-sudo: false
+sudo: required
 
 cache:
   directories:
     - $HOME/.composer/cache
 
 before_install:
+  - sudo apt-get update --yes
+  - sudo apt-get install --yes net-tools
```

Comment:
It is need for `netstat -n -t`. Without installing this package it is not working `command not found`. I can remove netstat call, but there can be small chance for reusing used port.